### PR TITLE
Update codeclimate badge in README

### PR DESCRIPTION
**Pull request by nporteschaikin**

### CODE REVIEW

**Comment by ABaldwinHunter** [comment]
FWIW I'd avoid the `channel/<name>` syntax for the future and reserve that format only for deployable channels - since they'll kick off a deploy.

LGTM

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Code Climate ESLint Engine
 
-[![Code Climate](https://codeclimate.com/repos/55841b7a6956801212006c92/badges/92d8261f1b6200f19af5/gpa.svg)](https://codeclimate.com/repos/55841b7a6956801212006c92/feed)
+[![Code Climate](https://codeclimate.com/repos/57f285a969744e0d830009a7/badges/5ac270c5078729d2b4e1/gpa.svg)](https://codeclimate.com/repos/57f285a969744e0d830009a7/feed)
```
Comment (ABaldwinHunter) [inline README.md]: can we use the OSS format of this link? E.g.: https://github.com/codeclimate/codeclimate-eslint/blob/master/README.md

**Comment by ABaldwinHunter** [comment]
Sorry one tiny change actually. Then LGTM

**Comment by dblandin** [comment]
Deleted the tag from DockerHub. Going to close this PR to prevent more accidental channel auto-deployments.

**Pull request closed without merging**